warcio/recordloader.py

```diff
@@ -55,7 +55,7 @@ class ArcWarcRecordLoader(object):
     NON_HTTP_SCHEMES = ('dns:', 'whois:', 'ntp:')
     HTTP_SCHEMES = ('http:', 'https:')
 
-    def __init__(self, verify_http=True, arc2warc=True):
+    def __init__(self, verify_http=True, arc2warc=True, fixup_bugs=True):
```

Comment:
In the spirit of how digest checking is handled by `ArchiveIterator`, `fixup_bugs=True` should probably default to `False`  
https://github.com/webrecorder/warcio/blob/759ab0777fb1ec03ec7f47bba270ac3f61e07a81/warcio/archiveiterator.py#L46

Reply:
The existing code did the bug fixup unconditionally, I'm preserving that default.

digest checking defaults off because it's expensive and @ikreymer prefers people remain in the dark as to how many invalid warcs there are :-)